packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -386,16 +386,21 @@ async function waitUntilMessage(
     const { exampleToTestMap } = getStats();
 
     for (const recordingId of newRecordingIds) {
-      await loadRecording(recordingId);
+      try {
+        await loadRecording(recordingId);
+      } catch (e) {
+        console.log(`Error during processing: ${e}`);
+      }
     }
 
     console.log("The following tests have been impacted by this change:");
     console.table(
       updatedExamples
         .map(example => {
           const tests = exampleToTestMap[example];
-
-          return ` • ${chalk.yellow(example)}${tests.map(test => `\n   • ${test}`).join("")}`;
+          return tests
+            ? ` • ${chalk.yellow(example)}${tests.map(test => `\n   • ${test}`).join("")}`
+            : "";
```

Comment:
Seems like we should log an explicit message in this case?

Reply:
Good idea; done.